scripts/monitoring/cron-send-security-updates-count.py

```diff
@@ -82,7 +82,7 @@ def get_security_updates(self):
             self.sec_updates = []
             raise subprocess.CalledProcessError(proc.returncode, subprocess.list2cmdline(cmd), output=stdout+stderr)
         lineslist = [line for line in stdout.split('\n') if line and 'Errno' not in line]
-        self.sec_updates = [UpdateItem(*re.split(r'\s+', line)) for line in lineslist]
+        self.sec_updates = [UpdateItem(*re.split(r'\s+', line)) for line in lineslist if re.split(r'\s+', line) == 3]
```

Comment:
You get lines like this:

Update notice RHEA-2017:1949 (from rhel-7-server-rpms) is broken, or a bad duplicate, skipping.
You should report this problem to the owner of the rhel-7-server-rpms repository.

In addition to the expected output of:
RHSA-2017:1865              Moderate/Sec.  libX11-1.6.5-1.el7.x86_64